package.json

```diff
@@ -1737,6 +1742,61 @@
                     ],
                     "description": "%vscode-docker.config.template.composeUp.description%"
                 },
+                "docker.commands.composeUpSubset": {
```

Comment:
Only good way I could work out to deal with putting `${profileList}` toward the beginning and `${serviceList}` at the end was to make a new customizable command.